internal/ingress/controller/nginx.go

```diff
@@ -1084,37 +1074,65 @@ ratio = {{ .OtelSamplerRatio }}
 parent_based = {{ .OtelSamplerParentBased }}
 `
 
+func datadogOpentracingCfg(cfg ngx_config.Configuration) (string, error) {
+	jsn := map[string]interface{}{
```

Comment:
I'd probably just call this `m` instead of `jsn`, especially since it's not json here, even if it becomes it later.

Reply:
will do